src/model/platform-setup/setup-mac.ts

```diff
@@ -110,17 +110,23 @@ class SetupMac {
     }
 
     const unityChangeset = await getUnityChangeset(buildParameters.editorVersion);
-    const moduleArgument = SetupMac.getModuleParametersForTargetPlatform(buildParameters.targetPlatform);
+    const moduleArguments = SetupMac.getModuleParametersForTargetPlatform(buildParameters.targetPlatform);
 
-    const command = `${this.unityHubExecPath} -- --headless install \
-                                          --version ${buildParameters.editorVersion} \
-                                          --changeset ${unityChangeset.changeset} \
-                                          ${moduleArgument} \
-                                          --childModules `;
+    const execArguments: string[] = [
+      '--',
+      '--headless',
+      'install',
+      ...['--version', buildParameters.editorVersion],
+      ...['--changeset', unityChangeset.changeset],
```

Comment:
prefer template string over spreading two strings.

Reply:
I wasn't sure if exec handles whitespaces of arguments correctly, but I found it's OK after investigating its source code. I'll revert changing `getModuleParametersForTargetPlatform` too.

Reply:
I see.